modules/apps_api/app/controllers/apps_api/v0/directory_controller.rb

```diff
@@ -11,7 +11,7 @@ class DirectoryController < ApplicationController
 
       def index
         render json: {
-          data: DirectoryApplication.order('LOWER(name)')
+          data: DirectoryApplication.order('LOWER(name)').paginate(page: params[:page], per_page: 10)
```

Comment:
Should per_page be pulled in from the request params?

Reply:
After talking to Tyler the number of 10 is correct via the acceptance criteria on the ticket, so this isn't an issue